plugins/inputs/kube_inventory/pod.go

```diff
@@ -82,15 +83,21 @@ func (ki *KubernetesInventory) gatherPodContainer(p corev1.Pod, cs corev1.Contai
 		fields["phase_reason"] = phaseReason
 	}
 
+	tok := strings.Split(c.Image, ":")
+
 	tags := map[string]string{
 		"container_name": c.Name,
+		"image":          c.Image,
```

Comment:
What is the `tok` variable above? Slice of container image + version? 
Does setting the image include the version as well? Or did you mean to do `tok[0]`?

Reply:
Thanks for pointing this out. I think I will just remove the version tag for now, since this can be extracted  from theimage tag.